funfact/lang/_terminal.py

```diff
@@ -165,13 +166,14 @@ def __init__(self, *size, symbol=None, initializer=None, optimizable=True):
         self.initializer = initializer
         self.optimizable = optimizable
 
-    def vectorize(self, n):
+    def vectorize(self, n, post):
```

Comment:
As above, maybe change `post` to `axis`?

Reply:
See reply above, we can change it, but I feel like a boolean is appropriate?